cmd/cost/get.go

```diff
@@ -179,10 +205,20 @@ func getOUsRecursive(OU *organizations.OrganizationalUnit, awsClient awsprovider
 }
 
 //Get cost of given account
-func getAccountCost(accountID *string, unit *string, awsClient awsprovider.Client, timePtr *string, cost *float64) error {
+func (o *getOptions) getAccountCost(accountID *string, unit *string, awsClient awsprovider.Client, cost *float64) error {
+
+	var start, end, granularity string
+	if o.time != "" {
```

Comment:
Is time mutually exclusive with start and end? If all three are set,  as far as I can read this, the start, end and granularity set here (lines 212-213) will be overwritten (lines 2127-219).

Reply:
The function from before is a check that errors out if the 3 flags are provided. So yes time and start/end are mutually exclusive